packages/headless/package.json

```diff
@@ -42,7 +42,8 @@
     "redux": "^4.0.5",
     "redux-mock-store": "^1.5.4",
     "redux-thunk": "^2.3.0",
-    "ts-debounce": "^2.0.1"
+    "ts-debounce": "^2.0.1",
+    "web-encoding": "^1.1.4"
```

Comment:
Provides `TextDecoder` so that headless is not depending on a global.

Ref: https://www.npmjs.com/package/web-encoding